src/dune/lib.ml

```diff
@@ -299,6 +299,7 @@ module T = struct
     ; unique_id : Id.t
     ; re_exports : t list Or_exn.t
     ; requires : t list Or_exn.t
+    ; requires_no_vlib : t list Or_exn.t
```

Comment:
It's the `requires` list with the virtual library filtered out in case this is an implementation of a virtual library. I added a comment in the code to reflect this.